multichain-testing/patches/starshipjs+2.0.0.patch

```diff
@@ -0,0 +1,35 @@
+diff --git a/node_modules/starshipjs/hooks.js b/node_modules/starshipjs/hooks.js
+index fb3e3cf..6b6567b 100644
+--- a/node_modules/starshipjs/hooks.js
++++ b/node_modules/starshipjs/hooks.js
+@@ -14,7 +14,7 @@ const useRegistry = async (configFile) => {
+     const registryUrl = `http://localhost:${config.registry.ports.rest}`;
+     const urls = [];
+     config.chains?.forEach((chain) => {
+-        urls.push(`${registryUrl}/chains/${chain.name}`, `${registryUrl}/chains/${chain.name}/assets`);
++        urls.push(`${registryUrl}/chains/${chain.id}`, `${registryUrl}/chains/${chain.id}/assets`);
```

Comment:
is this a bug in starship? if so, can we get a fix upstream?

Reply:
Yes, there was a breaking change in the 0.2.x helm chart release. I opened an issue: https://github.com/cosmology-tech/StarshipJS/issues/3